mailet/standard/src/test/java/org/apache/james/transport/matchers/FetchedFromTest.java

```diff
@@ -87,4 +87,15 @@ public void testHeaderWasRemovedAfterMatched() throws MessagingException {
 
         assertThat(mockedMail.getMessage().getHeader(FETCH_FROM_HEADER)).isNull();
     }
+
+    @Test
+    public void testHeaderWasNotRemovedIfNotMatched() throws MessagingException {
+        setupMockedMimeMessage(FETCH_FROM_HEADER, WRONG_HEADER_VALUE);
```

Comment:
usage of members is not a great thing, we don't have a clue what is tested from reading a test